src/i18n/fr.js

```diff
@@ -0,0 +1,93 @@
+export default {
+  error: {
+    forgotPassword: {
+      "too_many_requests": "Vous avez atteint la limite sur les tentatives de passe de réinitialisation. S'il vous plaît attendre avant de réessayer.",
+      "lock.fallback": "We're sorry, something went wrong when requesting the password change."
+    },
+    login: {
+      "blocked_user": "Utilisateur bloqué",
+      "invalid_user_password": "Courriel ou mot de passe incorrect.",
+      "lock.fallback": "We're sorry, something went wrong when attempting to log in.",
```

Comment:
> Nous sommes désolés, une erreur s'est produite lors de votre tentative de connection.

Not sure of connection is the best word for log in. @sandrinodimattia, @choopooly